sources/core/Stride.Core.Mathematics/Size2F.cs

```diff
@@ -122,6 +162,124 @@ public override int GetHashCode()
             return !left.Equals(right);
         }
 
+        /// <summary>
+        /// Implements the operator <c>/</c>, component wise.
```

Comment:
Description of a method shouldn't explain what they implement but what they do.

For instance: "Divides the first size by the second, component-wise".